test/DynamoCoreWpfTests/PythonNodeCustomizationTests.cs

```diff
@@ -24,13 +24,18 @@ public override void Open(string path)
             DispatcherUtil.DoEvents();
         }
 
-        protected override void GetLibrariesToPreload(List<string> libraries)
+        public override void Start()
         {
-            libraries.Add("DSCPython.dll");
-            libraries.Add("DSIronPython.dll");
+            base.Start();
```

Comment:
Can you remind me what does `Start()` do?

Reply:
a bunch of stuff, it basically starts the DynamoModel.